clients/go-core/gocore.sh

```diff
@@ -116,15 +123,15 @@ fi
 
 # Configure any mining operation
 if [ "$HIVE_MINER" != "" ] && [ "$HIVE_NODETYPE" != "light" ]; then
-    FLAGS="$FLAGS --mine --miner.coinbase $HIVE_MINER"
+    FLAGS="$FLAGS --mine --miner.corebase $HIVE_MINER --miner.threads=99"
```

Comment:
Are you sure we can handle 99 threads?

Reply:
I've tested it locally and it worked fine but I can set number of threads = 5 for example